gnuradio-runtime/lib/pmt/pmt_serialize.cc

```diff
@@ -239,10 +270,14 @@ static bool deserialize_untagged_u64_vector(std::vector<uint64_t>& data,
 {
     std::streambuf::traits_type::int_type t;
     data.resize(nitems);
-    t = sb.sgetn((char*)&data[0], nitems * sizeof(uint64_t));
+    std::vector<uint8_t> buf(nitems * sizeof(uint64_t));
+    t = sb.sgetn((char*)&buf[0], nitems * sizeof(uint64_t));
     sb.pubseekoff(sizeof(uint64_t) * nitems, std::ios_base::cur);
     for (size_t i = 0; i < nitems; i++) {
-        boost::endian::big_to_native_inplace(data[i]);
+        data[i] = ((uint64_t)buf[8 * i] << 56) | ((uint64_t)buf[8 * i + 1] << 48) |
```

Comment:
For code cleanliness, should we make this bit of code an inline function?  it is reused several times.